README.md

```diff
@@ -0,0 +1,14 @@
+Project Title:
```

Comment:
The README is a really important part of your repo, and this is missing all the important info. Some things that would make it much better:
- A description of what the app actually does and what its features are
- installation instructions (especially regarding the database and the need to create environment variables),
- what technologies are required to use it, what the application dependencies are (`Express` and `MySQL2`), and what the development dependencies are (`nodemon`, `dotenv`, `mocha`, `chai`, `superTest`)
- Its an opportunity to show off what youve built! Include docker and PostGres as included technologies
- information about the routes provided by the app, what type of HTTP verb they all require and an example request body would be a great addition,
- how to test the app,
- what you would do if you had more time (if applicable)

They should be written in markdown language, which is very simple. Heres a helpful cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/)